ontohgis.ttl

```diff
@@ -5311,8 +5123,8 @@ ontohgis:administrative_type_111 rdf:type owl:Class ;
                                  ontohgis:isSymbolisedByMeansOf ontohgis:map_symbol_30 ;
                                  rdfs:isDefinedBy ontohgis:administrative_system_19 ;
                                  rdfs:label "Kreis"@de ,
-                                            "district"@en ,
-                                            "powiat"@pl ,
+                                            "district (Silesia (1372-1782))"@en ,
+                                            "powiat (Śląsk (1372-1782))"@pl ,
                                             "terra"@la ;
```

Comment:
Why wasn't the Latin label updated alongside the others?

Reply:
> We have no Latin names for administrative system, at this moment. The same is with label @de. What do you suggest?

http://purl.org/ontohgis#administrative_type_111 has the Latin label "terra"@la.

Reply:
Yes, but we have no Latin term for the administrative system which is going to be a part of label for administrative type. We have mostly pl and en labels for administrative systems